code/modules/mob/living/simple_animal/friendly/diona_nymph.dm

```diff
@@ -205,6 +205,8 @@
 	adult.real_name = adult.name
 	adult.ckey = ckey
 	adult.real_name = adult.dna.species.get_random_name()	//I hate this being here of all places but unfortunately dna is based on real_name!
+	// Doesn't allow cult or vampires to benefit from them fully, similar to "TRAIT_HOLY".
```

Comment:
this comment is a bit misleading, because its not like the holy trait, it just prevents conversion or blood gain. a better comment would be something like "doesnt allow conversion into cults or usable blood gain for vampires"